cmake/Modules/GrSwig.cmake

```diff
@@ -118,12 +118,8 @@ macro(GR_SWIG_MAKE name)
     list(INSERT GR_SWIG_INCLUDE_DIRS 0 "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>")
     list(INSERT GR_SWIG_INCLUDE_DIRS 0 "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>")
 
-    if (PYTHON3)
-        set(py3 "-py3")
-    endif (PYTHON3)
-
     #setup the swig flags with flags and include directories
-    set(CMAKE_SWIG_FLAGS -fvirtual -modern -keyword -w511 -w314 -relativeimport ${py3} -module ${name} ${GR_SWIG_FLAGS})
+    set(CMAKE_SWIG_FLAGS -fvirtual -modern -keyword -w511 -w314 -relativeimport -module ${name} ${GR_SWIG_FLAGS})
```

Comment:
So this removes the `-py3` flag. Is that intentional?

Reply:
huh. no, not actually, but: the testcases _work_ without the flag.

So. Maybe. yes?

Reply:
Hmmm ... SWIG3 docs on Py3 are < http://swig.org/Doc3.0/SWIGDocumentation.html#Python_python3support > ... doesn't say that `-py3` is a default, so I'm guessing if we don't include it then SWIG tries to generate code that is compliant with both Py2 and Py3. Adding in the flag adds Py3-only features ... so, thinking to just use the flag outright here is the right thing to do.

Reply:
no. no, removing that is not OK: `-py3` isn't documented via `--help`, or `man swig`, but it seems to [_do_](http://www.swig.org/Doc3.0/SWIGDocumentation.html#Python_python3support) things. So, adding this back.

Reply:
`swig -python -help` shows me the `-py3` flag ...

Reply:
Fixed on the other PR.